binary_and_decimal.rb

```diff
@@ -4,64 +4,99 @@
 #    number using the algorithm you devised in class. Print out the decimal value.
 def binary_to_decimal
   puts "Not implemented"
-end
+binary = []
+  8.times do
+    binary << rand(0...2)
+  end
 
-# 2. Write a method named `array_equals` that accepts two integer arrays as
-#    parameters. The method return `true` if the arrays contain the same elements
-#    in the same order, and returns `false` otherwise.
-# Note: Do not use Array class methods for comparing the whole array at once.
-#       You may use array indexing to retrieve one element at a time,
-#       compare individual elements with each other and
-#       you may retrieve the length of an array.
-def array_equals(array1, array2)
-  puts "Not implemented"
-  return true
+  binary_as_string = ""
+  binary.each do |num|
```

Comment:
You are not converting the 8 digit binary number to decimal anywhere in this method. The problem statement ask for you create a 8 digit binary number and convert it to decimal.